packages/discord.js/src/structures/Role.js

```diff
@@ -188,8 +188,14 @@ class Role extends Base {
    * @readonly
    */
   get position() {
-    const sorted = this.guild._sortedRoles();
```

Comment:
Doesn't this mean we no longer need _sortedRoles? 👀 🧹

Reply:
It is used in setting position

Reply:
We still do:

https://github.com/discordjs/discord.js/blob/b327f4925ff557e0aa8273d7f017aa616226ca06/packages/discord.js/src/managers/RoleManager.js#L252